### ISSUE TITLE
Authority browse result zero

### ISSUE THREAD

**Issue by viola-v** [dag-hammarskjold-library/dlx-rest#1217]
I think this came up before but I don't remember what the issue was - there's a record in the authorities browse with zero bibs, meaning the record doesn't exist? Why is it there?
![image](https://github.com/dag-hammarskjold-library/dlx-rest/assets/110108422/7a35c723-2c19-407f-8ba6-584f65ae3a92)

Search doesn't find it: 
![image](https://github.com/dag-hammarskjold-library/dlx-rest/assets/110108422/51b64c26-d6f1-42a7-8c78-164ad6617cd4)

**Comment by viola-v** [comment]
Just confirming that this issue persists, so it wasn't related to the logical fields being rebuilt...

**Comment by viola-v** [comment]
Not sure if we ever did anything about this, but it just happened again:

![image](https://github.com/user-attachments/assets/3d681905-9ca6-43b8-8682-73264a5b4b81)

**Comment by jbukhari** [comment]
> Not sure if we ever did anything about this, but it just happened again:
> 
> ![image](https://private-user-images.githubusercontent.com/110108422/348269386-3d681905-9ca6-43b8-8682-73264a5b4b81.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjA4MDg2NTQsIm5iZiI6MTcyMDgwODM1NCwicGF0aCI6Ii8xMTAxMDg0MjIvMzQ4MjY5Mzg2LTNkNjgxOTA1LTljYTYtNDNiOC04NjgyLTczMjY0YTViNGI4MS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNzEyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDcxMlQxODE5MTRaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0xZDFhMGQyYTVmZmUyYmY4MDgxYzg5ZjYyNDAxNTBhYjE0ZDU2M2E0NWM1Y2RkMTIwN2M5MjFjYmFlMmQ5ODIwJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.XVnltxAKb2Q3RBTya6MWUJWZ_p4vLz1lkDugFixn6GY)

@viola-v I am not currently seeing that when I run that search

**Comment by viola-v** [comment]
@jbukhari Sorry, that was the wrong screenshot, this is the correct one for this issue: 
![image](https://github.com/user-attachments/assets/401ffead-a1e7-4962-a5ed-d62a71324676)

**Comment by jbukhari** [comment]
> Re-opening this ticket, as I'm seeing this issue again, for example: <img>

Those values are not in the browse list anymore. @aaronhelton maybe the build logical fields script has been run since then?

**Comment by aaronhelton** [comment]
Yes, I mentioned this in our meeting. I believe I asked whether that script would fix this. And while we can run the script periodically, I'm sure we want to know what's causing the issue in the first place.

**Comment by JoelleSciboz** [comment]
Annother case, this might help you to figure out the issue: 

<img width="307" alt="Image" src="https://github.com/user-attachments/assets/0a46e1f9-8783-4a44-bf5c-c7c12349e758" />

**Comment by jbukhari** [comment]
@aaronhelton From the history collection, we can see the value "**A/79/6 (Sect. 3)/Add.7**" should have been deleted from the _index_symbol collection during the update at `2024-10-15T12:06:44.304+00:00`. Is it possible to check the logs from that time to see if there was any error?

<img width="475" height="215" alt="Image" src="https://github.com/user-attachments/assets/1a4ab2db-bada-4e43-9335-ff9d5f42a684" />
